CustomUIButton.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomUIButton.swift
+//  Connnect
+//
+//  Created by Stella Su on 11/22/16.
+//  Copyright © 2016 iOS-Connect. All rights reserved.
+//
+
+import Foundation
```

Comment:
`import UIKit` pulls in `Foundation` as well so we could leave off this `import Foundation`